docs/_entities/reshare.md

```diff
@@ -2,15 +2,25 @@
 title: Reshare
 ---
 
-This entity represents a reshare of a [StatusMessage][status_message]. It inherits from [Post][post].
+This entity represents a reshare of a [StatusMessage][status_message].
+
+{% include warning_box.html
+   title="Future of reshares"
+   content="<p>Current versions of diaspora* handle reshares like they inherit from
+   <a href=\"/diaspora_federation/entities/post.html\">Post</a> and allow interactions (Comments and
+   Likes) with the reshare. In the future the reshare entity will only be used to increase the spread of a
+   Post, more information about that can be found in
+   <a href=\"https://github.com/diaspora/diaspora_federation/issues/83\">this issue</a>.</p>
+   <p>There is currently also a special case for reshares with a deleted root post. It is valid when the
+   reshare doesn't include <code>root_author</code> and <code>root_guid</code>. If there is only one of
+   <code>root_author</code> and <code>root_guid</code> is present, then the entity is not valid. Once
+   reshares are only used to increase the reach of a post, reshares without <code>root_author</code> and
+   <code>root_guid</code> are no longer valid, reshares will be deleted when the original post is deleted.</p>"
```

Comment:
`... will no longer be valid and reshares will [...] if the original post [...]`